hummingbird/ml/convert.py

```diff
@@ -27,14 +27,41 @@
 from .supported import lgbm_operator_list  # noqa
 
 
+def _is_onnx_model(model):
+    """
+    Function returning whether the input model is an ONNX model or not.
+    """
+    return "onnx" in model.__module__ and "graph" in dir(model)
```

Comment:
maybe something like

```
>>> type(onnx_ml_model).__name__
'ModelProto'
```
or
```
>>> isinstance(onnx_ml_model, onnx.onnx_ONNX_REL_1_6_ml_pb2.ModelProto)
True
```

Reply:
i guess for the second one, the version isn't known. but we could check just the first part

Reply:
Yes the version can change. Should I go with the first one?

Reply:
yah, i think the first one is cleanest, although i don't feel strongly :)